transforms/language/docling2parquet/README.md

```diff
@@ -66,8 +66,8 @@ The transform can be initialized with the following parameters.
 | `artifacts_path`             | <unset> | Path where to Docling models artifacts are located, if unset they will be downloaded and fetched from the [HF_HUB_CACHE](https://huggingface.co/docs/huggingface_hub/en/guides/manage-cache) folder. |
 | `contents_type`         | `text/markdown`        | The output type for the `contents` column. Valid types are `text/markdown`, `text/plain` and `application/json`. |
 | `do_table_structure`         | `True`        | If true, detected tables will be processed with the table structure model. |
-| `do_ocr`                     | `True`        | If true, optical character recognition (OCR) will be used to read the content of bitmap parts of the document. |
-| `ocr_engine`                 | `easyocr`     | The OCR engine to use. Valid values are `easyocr`, `tesseract`, `tesseract_cli`. |
+| `docling2parquet_do_ocr`                     | `True`        | If true, optical character recognition (OCR) will be used to read the content of bitmap parts of the document. |
```

Comment:
The fact of prepending `docling2parquet_` to the parameters depend on how the transform is used. In some use cases you don't need it, in others you need it.

I don't think we want to update the description of the parameters here. (in case, it must be for *all* of them)
Just a few lines below, it is described that in the CLI usage the user must prepend the parameters with `docling2parquet_`.

We should describe better all the usages when it is needed since it is not only the CLI.
@shahrokhDaijavad @touma-I this may apply to many other transforms.